python_modules/dagster/dagster/_core/storage/pipeline_run.py

```diff
@@ -524,20 +529,40 @@ def tags_for_backfill_id(backfill_id: str) -> Mapping[str, str]:
     def tags_for_partition_set(
         partition_set: "PartitionSetDefinition", partition: "Partition"
     ) -> Mapping[str, str]:
-        from dagster._core.definitions.multi_dimensional_partitions import (
-            MultiPartitionKey,
-            get_tags_from_multi_partition_key,
-        )
-
         tags = {PARTITION_SET_TAG: partition_set.name}
-        if isinstance(partition.name, MultiPartitionKey):
-            tags.update(get_tags_from_multi_partition_key(partition.name))
-        else:
-            tags[PARTITION_NAME_TAG] = partition.name
-
+        tags.update(get_tags_from_partition_key(partition.name))
         return tags
 
 
+def get_tags_from_partition_key(partition_key: str) -> Mapping[str, str]:
```

Comment:
What do you think about making this a method of `PartitionsDefinition` that gets overriden by `MultiPartitionsDefinition`?

Reply:
sounds good, good call!